PowerShell/MineStat/MineStat.psm1

```diff
@@ -124,10 +124,21 @@ function MineStat {
     hidden [string]$connection_status = [ConnStatus]::Unknown
 
     ServerStatus($address, $port, $timeout, [SlpProtocol]$queryprotocol) {
-      $this.address = $address
-      $this.port = $port
+      try {
+        $resolved = Resolve-DnsName -type srv _minecraft._tcp.$address -ErrorAction Stop

```

Comment:
Should we also perform a check: if `BedrockRaknet` is specified, then use `_minecraft._udp.$address`?

Reply:
No, I don't think Minecraft Bedrock supports SRV records, as there is an open feature request for that: https://feedback.minecraft.net/hc/en-us/community/posts/360016429012-SRV-Record-Support-Bedrock-Edition-

Reply:
I am fine with excluding SRV resolution for the UDP-based service. I was not really thinking of the Bedrock client in this context, but rather simply adding the functionality to our project for convenience (since DNS behaves independently of the client and our project) and for future-proofing. We can revisit this item if and when the feature request that you linked is implemented by Microsoft/Mojang.